integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -287,6 +287,7 @@ def test_mod(data_gen):
                               DecimalGen(precision=37, scale=0), DecimalGen(precision=37, scale=10),
                               _decimal_gen_7_7]
 
+@allow_non_gpu("ProjectExec", "Pmod")
```

Comment:
Instead of saying it is okay for all of the tests to fall back to the CPU and we don't care. Can we split the decimal parts off and verify that we fell back to the CPU?